lib/pages/home/widgets/stories_list.dart

```diff
@@ -5,18 +5,21 @@ import 'package:responsive_framework/responsive_wrapper.dart';
 import 'story_circle.dart';
 
 class StoriesList extends StatelessWidget {
+  const StoriesList({super.key});
+
   @override
   Widget build(BuildContext context) {
     final mobile = ResponsiveWrapper.of(context).isMobile;
+
     return Container(
       height: 110,
       margin: EdgeInsets.only(top: mobile ? 15 : 30),
       child: ListView.separated(
         scrollDirection: Axis.horizontal,
         padding: const EdgeInsets.symmetric(horizontal: 16),
         itemCount: 15,
-        separatorBuilder: (_, __) => SizedBox(width: 10),
-        itemBuilder: (_, i) => StoryCircle(),
+        separatorBuilder: (_, __) => const SizedBox(width: 10),
+        itemBuilder: (_, i) => const StoryCircle(),
```

Comment:
pass `__` instead `i`, because is not used and to be equals to `separatorBuilder` and follow convention.